common/djangoapps/track/shim.py

```diff
@@ -83,6 +83,10 @@ def remove_shim_context(event):
     'edx.video.seeked': 'seek_video',
     'edx.video.transcript.shown': 'show_transcript',
     'edx.video.transcript.hidden': 'hide_transcript',
+    'edx.video.closed_captions.shown': 'show_closed_captions',
+    'edx.video.closed_captions.hidden': 'hide_closed_captions',
+    'edx.video.language_menu.shown': 'video_show_cc_menu',
+    'edx.video.language_menu.hidden': 'video_hide_cc_menu'
 }
```

Comment:
Ok. Thanks for the explanation.  It wasn't clear from the first comment if that was the expected behavior.